app/stores/UsersStore.js

```diff
@@ -25,6 +25,11 @@ export default class UsersStore extends BaseStore<User> {
     return filter(this.orderedData, user => user.isSuspended);
   }
 
+  @computed
+  get activeOrInvited(): User[] {
```

Comment:
`active()` filters out invited users too, and we want them to be shown on these screens so admins can organize people before they join.